### PR TITLE
remove dead code

### PR DESCRIPTION
**Pull request by jessfraz**
so I ran https://github.com/remyoudompheng/go-misc/blob/master/deadcode/deadcode.go
on docker just to see and cleaned some things up

lets see what breaks

### CODE REVIEW

**Comment by jfrazelle** [comment]
thx @GordonTheTurtle

**Comment by calavera** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

File: daemon/volumes_unix.go [hunk 1]
```diff
@@ -70,53 +68,6 @@ func sortMounts(m []execdriver.Mount) []execdriver.Mount {
 	return m
 }
 
-// migrateVolume links the contents of a volume created pre Docker 1.7
-// into the location expected by the local driver.
-// It creates a symlink from DOCKER_ROOT/vfs/dir/VOLUME_ID to DOCKER_ROOT/volumes/VOLUME_ID/_container_data.
-// It preserves the volume json configuration generated pre Docker 1.7 to be able to
-// downgrade from Docker 1.7 to Docker 1.6 without losing volume compatibility.
-func migrateVolume(id, vfs string) error {
```
Comment (cpuguy83) [inline daemon/volumes_unix.go]: I don't think we should have removed this, but the fact that it's dead is likely the cause that a lot of people have had issues upgrading to 1.10... not sure where these were being called from before.

ping @calavera

**Pull request merged**